src/System.Data.SqlClient/tests/ManualTests/SQL/SqlBulkCopyTest/Transaction4.cs

```diff
@@ -31,7 +31,11 @@ public static void Test(string srcConstr, string dstConstr, string dstTable)
                             conn3.Open();
                             // Start a local transaction on the wrong connection.
                             SqlTransaction myTrans = conn3.BeginTransaction();
+#if uapaot // Reflection is blocked for internal members on uapaot
```

Comment:
Rather than all these ugly #if's sprinkled around, what about updating your AssertThrowsWrapper instead - to only consult the error message when it's not in net native? 

Analogous to what we already do with parameter names:
https://github.com/dotnet/corefx/blob/master/src/CoreFx.Private.TestUtilities/src/System/AssertExtensions.cs#L38

```c#
            if (!RuntimeInformation.FrameworkDescription.StartsWith(".NET Native"))
                Assert.Equal(expectedParamName, exception.ParamName);
```

That keeps the net native bit in one place.